# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 06f7b45d044a
# Full Hash: 06f7b45d044ac11b4b34e6bb61c10555998952e6
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 09:10:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 5 - Implement support for COLRv1 glyphs represented as acyclic graphs of paint records. r=gfx-reviewers,lsalzman
#   
#   Depends on D153870
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152041
# ==============================================================================

diff -r 02d6294b7a6d -r 06f7b45d044a modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Aug 16 12:48:33 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Aug 16 12:48:33 2022 +0000
@@ -5729,6 +5729,11 @@
     mirror: always
 #endif
 
+- name: gfx.font_rendering.colr_v1.enabled
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+
 - name: gfx.font_rendering.opentype_svg.enabled
   type: RelaxedAtomicBool
   value: true
